Tests/L0/Loc/_suite.ts

```diff
@@ -8,6 +8,7 @@ import path = require('path');
 import fs = require('fs');
 
 describe('Loc String Suite', function() {
+	this.timeout(20000);
```

Comment:
One of the loc tests (grep ps1/psm1 files for missing strings) timed out at two seconds. It doesn't appear that the override at the individual test level actually does anything.